go/libkb/keyfamily.go

```diff
@@ -305,11 +308,8 @@ func (ckf ComputedKeyFamily) InsertEldestLink(tcl TypedChainLink, username Norma
 
 // ParseKeyFamily takes as input a dictionary from a JSON file and returns
 // a parsed version for manipulation in the program.
-func ParseKeyFamily(jw *jsonw.Wrapper) (ret *KeyFamily, err error) {
-	G.Log.Debug("+ ParseKeyFamily")
-	defer func() {
-		G.Log.Debug("- ParseKeyFamily -> %s", ErrToOk(err))
-	}()
+func ParseKeyFamily(g *GlobalContext, jw *jsonw.Wrapper) (ret *KeyFamily, err error) {
+	g.Trace("ParseKeyFamily", func() error { return err })()
```

Comment:
How does this work? It looks like it logs the exit line immediately instead of deferring it?